crates/recorder/src/benchmark.rs

```diff
@@ -128,13 +128,19 @@ impl<'a, 'b> Instance<'a, 'b> {
     }
 }
 
+pub struct BenchParameters {
+    pub must_instantiate: bool,
+    pub must_execute: bool,
+}
```

Comment:
Having `must_instantiate == false && must_execute == true` is a bit nonsensical; it would be nice if the type couldn't represent invalid states. Therefore, I think it would be nicer if we had something like the following:

```rust
use sightglass_data::Phase;

pub fn benchmark(
    // existing params...

    // Optionally stop after the given phase, rather than running all phases.
    stop_after: Option<Phase>,
) -> Result<()> {
    // ...
}
```

Reply:
Good call; even allowing the combination of CLI flags `--execute true --instantiate false` was a bit misleading, so I've changed it so that one directly pass the `stop_after` Phase option to the command line.